scripts/ubuntu/hello-world-deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+export DEBIAN_FRONTEND=noninteractive
+
+apt update
```

Comment:
How do you know you have to run this? When you do this on like 1000s of machines it's actually something that takes a long time. What is the check to see if you need to run this? What is the desired state you want? Maybe you only need to run this if something else isn't true?